src/coreclr/System.Private.CoreLib/src/System/TypedReference.CoreCLR.cs

```diff
@@ -16,6 +16,17 @@ public ref partial struct TypedReference
         private readonly ref byte _value;
         private readonly IntPtr _type;
 
+        // implementation of CORINFO_HELP_GETREFANY
+        internal static ref byte GetRefAny(IntPtr clsHnd, TypedReference typedByRef)
```

Comment:
```suggestion
        [StackTraceHidden]
        internal static ref byte GetRefAny(IntPtr clsHnd, TypedReference typedByRef)
```
to match other similar helpers https://github.com/dotnet/runtime/blob/main/src/libraries/System.Private.CoreLib/src/System/Math.cs#L1672